lua/astrocommunity/file-explorer/fyler-nvim/init.lua

```diff
@@ -1,15 +1,53 @@
 return {
-  "A7Lavinraj/fyler.nvim",
-  dependencies = {
-    "echasnovski/mini.icons",
-    {
-      "AstroNvim/astrocore",
-      opts = function(_, opts)
-        local maps = assert(opts.mappings)
-        maps.n["<Leader>y"] = { "<Cmd>Fyler<CR>", desc = "Fyler" }
-      end,
+  {
+    "nvim-neo-tree/neo-tree.nvim",
+    enabled = false,
+  },
+
+  {
+    "A7Lavinraj/fyler.nvim",
+    dependencies = {
+      "echasnovski/mini.icons",
+      {
+        "AstroNvim/astrocore",
+        opts = function(_, opts)
+          local maps = opts.mappings or {}
+          maps.n["<Leader>e"] = {
+            function() require("fyler").toggle { kind = "float" } end,
+            desc = "Open with fyler (floating)",
+          }
+          opts.mappings = maps
+        end,
+      },
+    },
+
+    opts = {
+      close_on_select = true,
+      confirm_simple = false,
+      default_explorer = true,
+
+      win = {
+        border = "rounded",
+        kind = "replace",
+      },
+
+      git_status = {
+        enabled = true,
+        symbols = {
+          Modified = "●",
+          Added = "✚",
+          Deleted = "✖",
+          Untracked = "★",
+        },
+      },
+
```

Comment:
```suggestion
```

The default for this I think is fine i think, we can remove this so it matches what is provided

```
  -- Git integration
  git_status = {
    enabled = true,
    symbols = {
      Untracked = "?",
      Added = "+",
      Modified = "*",
      Deleted = "x",
      Renamed = ">",
      Copied = "~",
      Conflict = "!",
      Ignored = "#",
    },
  },
```